src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -267,10 +267,11 @@ private void configureCountryImageView() {
 
     countryImageView.setVisible(true);
 
-    Tooltip countryTooltip = new Tooltip(player.getCountry());
-    countryTooltip.textProperty().bind(player.countryProperty());
-
-    Tooltip.install(countryImageView, countryTooltip);
+    if (!StringUtils.isEmpty(player.getCountry())) { // empty (non-null) string -> empty tooltip
+      i18n.getCountryNameLocalized(player.getCountry()).ifPresent(countryName -> //
```

Comment:
Please don't pollute our code with abused end-of-line comments only required because of a shitty Eclipse formatter configuration 😁 
Using IntelliJ is strongly recommended.

Generally, we don't use end-of-line comments but instead put them in the line before.
That being said, I'm not even sure what the comment is trying to say so I'd rather just remove it (we only comment "why" something is done, never "what" is done)